client/src/Features.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import About from './About'
+
+export class Features extends Component {
```

Comment:
If you aren't using state in your component, you can use a functional component:
```js
function Features(props) {
  return(
  // your jsx here
  )
}
```